app/main.py

```diff
@@ -2,0 +1,37 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self._hidden: bool = False
+        Animal.alive.append(self)
+
+    @property
+    def hidden(self) -> bool:
+        return self._hidden
+
+    def __repr__(self) -> str:
```

Comment:
Code Style: Use interpolation instead of concatenation. Python's f-string formatting (also called interpolated string literals) is a more readable, more concise way to include expressions inside string literals. So, you should use f-string formatting instead of string concatenation.